code/game/machinery/computer/security.dm

```diff
@@ -178,10 +186,14 @@
 						if("execute")
 							if((access_magistrate in authcard_access) || (access_armory in authcard_access))
 								active2.fields["criminal"] = "*Execute*"
-								message_admins("[key_name_admin(usr)] authorized execution for [their_rank] [their_name], with comment: [t1]")
+								message_admins("[key_name_admin(usr)] authorized <span class='warning'>EXECUTION</span> for [their_rank] [their_name], with comment: [visible_reason] \

```

Comment:
Instead of all of these separate manually-constructed admin buttons, just use `[ADMIN_FULLMONTY(usr)] authorized`